dev/dependencyList

```diff
@@ -79,6 +81,7 @@ metrics-core/4.2.8//metrics-core-4.2.8.jar
 metrics-jmx/4.2.8//metrics-jmx-4.2.8.jar
 metrics-json/4.2.8//metrics-json-4.2.8.jar
 metrics-jvm/4.2.8//metrics-jvm-4.2.8.jar
+mysql-connector-java/8.0.27//mysql-connector-java-8.0.27.jar
```

Comment:
We can not add this to binary release because of license

Reply:
thanks